INN/umbrella-currentorg
# Single project view

## Changes

This pull request makes the following changes:

- Adds a way to link to single post by using a `project_id=ID` query param
- Pulls in single project data via AJAX and displays it in the single project holder partial

![screencapture-currentorg-test-local-projects-2020-05-29-12_35_40](https://user-images.githubusercontent.com/18353636/83283312-0560db00-a1a9-11ea-8ee8-bfcd9c89f191.png)

## Why



For #125

## Testing/Questions

Features that this PR affects:

- Single project view
- Functionality for linking to a specific project


Questions that need to be answered before merging:

- [ ] Does anything need to be sanitized better?
- [ ] Any safety check conditionals missing?
- [ ] Is there a better way to do any of this?
- [ ] When do we get to style this?
- [ ] What issue pertains to the mobile version?

Steps to test this PR:

1. Create a project with all possible meta fields filled out
2. View the project on the frontend and make sure everything displays without issue
3. Make sure the unique project link works and isn't broken
